### PR TITLE
fix(abstract-cosmos): allow valueless transfer for cosmos coin

### PR DESCRIPTION
**Pull request by ajays97**
Ticket: BG-78370, WP-423

### CODE REVIEW

**Review by OttoAllmendinger — DISMISSED** [review]

File: modules/abstract-cosmos/src/cosmosCoin.ts [hunk 1]
```diff
@@ -126,6 +126,10 @@ export class CosmosCoin extends BaseCoin {
     throw new Error('Method not implemented.');
   }
 
+  valuelessTransferAllowed(): boolean {
+    return true;
+  }
```
Comment (mukeshsp) [inline modules/abstract-cosmos/src/cosmosCoin.ts:131]: @ajays97 it should be in osmo module.

Comment (DinshawKothari) [reply modules/abstract-cosmos/src/cosmosCoin.ts:131]: yes.

Comment (DinshawKothari) [reply modules/abstract-cosmos/src/cosmosCoin.ts:131]: Mark it false here, and true in osmo module.

**Review by DinshawKothari — CHANGES_REQUESTED** [review]

File: modules/abstract-cosmos/src/cosmosCoin.ts (hunk 1 above)
Comment (ajays97) [reply modules/abstract-cosmos/src/cosmosCoin.ts]: Makes sense. Adding it to only osmo for now.

**Review by mukeshsp — APPROVED** [review]

**Review by Vijay-Jagannathan — APPROVED** [review]

**Review by DinshawKothari — APPROVED** [review]

**Pull request merged**